docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -179,6 +179,11 @@ There are some config options that affect this:
 
 include::{generated-dir}/config/quarkus-jacoco-jacoco-config.adoc[opts=optional, leveloffset=+1]
 
+[TIP]
+====
+When working with a multi-module project, then for code coverage to work properly, the dependent modules need to be properly xref:cdi-reference.adoc#bean_discovery[indexed].
```

Comment:
Is it really "dependent"? I'd say it's more like "upstream" or "the modules the tests depend upon" (or something less bulky).

Reply:
Changed to `upstream`